engine_modules/corporation/models.py

```diff
@@ -12,6 +14,9 @@
 from engine_modules.market.models import Market
 from engine.models import Game
 
```

Comment:
non \n between module imports, except to split "system import", "django import" and "user import"